### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

**Comment by peter-edge** [comment]
Note that the `yarpc-protobuf-raw-response` header key changed to `rpc-protobuf-raw-response`, I deleted this per @breerly's suggestion

**Comment by abhinav** [comment]
> Note that the yarpc-protobuf-raw-response header key changed to rpc-protobuf-raw-response, I deleted this per @breerly's suggestion

Changed comment to just that the wire repr. of proto changed and that this will break everyone.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,28 +4,35 @@ Releases
...
 -   x/roundrobin: Added support for taking peer list updates before and after
     the peer list has been started.
--   Moved the RoundRobin Peer List out of the /x/ package.
--   Fix race conditions in hostport.Peer.
 -   x/config: Fix bug where embedded struct fields could not be interpolated.
 -   x/config: Fix bug where Chooser and Updater fields could not be interpolated.
--   Pool inbound thrift request buffers to reduce allocations.
--   x/grpc: Remove NewInbound and NewSingleOutbound in favor of
-    functions on Transport.
+-   x/grpc: Remove `NewInbound` and `NewSingleOutbound` in favor of functions
```
Comment (kriskowal) [inline CHANGELOG.md]: functions/methods.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,28 +4,35 @@ Releases
 v1.9.0-dev (unreleased)
 -------------------
 
+-   Different encodings can now register handlers under the same procedure
+    name.
```
Comment (kriskowal) [inline CHANGELOG.md]: Nice catch.

**Pull request merged**